app/src/main/res/values/styles.xml

```diff
@@ -13,6 +13,19 @@
         <item name="windowNoTitle">true</item>
     </style>
 
+    <style name="One_line">
+        <item name="android:ellipsize">end</item>
+        <item name="android:maxLines">1</item>
+        <item name="android:singleLine">true</item>
+    </style>
+
+    <style name="Default_margins">
```

Comment:
You've grouped styles by probably not the best characteristics. What will you do if you need a single line text with default margins (you can apply only one style per view)?

Usually styles are grouped by views. For example, big black text with some margins for header or 30 dp rounded ImageView for avatars.